driver/ppm_flag_helpers.h

```diff
@@ -34,6 +34,13 @@ or GPL2.txt for full copies of the license.
 #define PPM_MS_MGC_MSK 0xffff0000
 #define PPM_MS_MGC_VAL 0xC0ED0000
 
+/* Check if the res is different from `PPM_SUCCCES` */
```

Comment:
```suggestion
/* Returns the res if it's different than PPM_SUCCESS */
```
I think this better describes the semantics. WDYT?

If you agree, I would also suggest renaming the macro to something more meaningful (naive proposal: `RETURN_UNSUCCESSFUL_RES`)

Reply:
Agree on the comment is clearer, thank you :rocket: Also on the name but probably I would highlight the fact that we directly return inside the macro, what about `RETURN_IF_UNSUCCESSFUL_RES`? Don't know if it is clearer or not :thinking:

Reply:
@jasondellaluce WDYT about this?

Reply:
Yeah that works for me!